test/integration/router_test.go

```diff
@@ -526,6 +533,9 @@ func TestRouterPathSpecificity(t *testing.T) {
 	if err := waitForRoute(routeAddress, "www.example.com", "http", nil, tr.HelloPod); err != nil {
 		t.Fatalf("unexpected response: %q", err)
 	}
+	if err := waitForRoute(routeTestAddress, "www.example.com:80", "http", nil, tr.HelloPodPath); err != nil {
```

Comment:
I'm not sure I understand. These new lines should be making requests against the path route. That's why it's testing against `routeTestAddress` instead of `routeAddress`. It looks like I should probably be using `getRoute()` instead of `waitForRoute()`, but that's a minor thing that I can fix.